src/Bot/Log.hs

```diff
@@ -49,6 +55,18 @@ recordUserMsg sender msg = do
         }
   return ()
 
+getRecentLogs :: Seconds -> Effect [LogRecord]
+getRecentLogs offset = do
+  currentTime <- now
+  let diff = intToNominalDiffTime $ -1 * offset
+  let startDate = addUTCTime diff currentTime
+  -- TODO there should be comparing selector when implemented
```

Comment:
Please format your unreported TODOs according to this https://github.com/tsoding/snitch#unreported-todo The colon after after `TODO` is mandatory.